pdr_backend/lake/table.py

```diff
@@ -73,12 +75,46 @@ def drop_tables_from_st(db: DuckDBDataStore, type_filter: str, st: UnixTimeMs):
     logger.info("truncated %s rows from %s tables", trunc_count, table_count)
 
 
+def get_caller_ppss():
+    caller_ppss = None
+    prev_frame = inspect.currentframe()
```

Comment:
Hi calina, 

I do not think it's best practice to be going through the call stack in this way to retrieve data.... I have never seen this implemented anywhere for functional/engineering purposes.

**[Separating Concerns]**
The lake cli commands expect "fixed date values" rather than "fuzzy, interpretable dates"... the following implementation to execute cli commands addresses this exactly as required, without adding any additional requirements or complexity.
https://github.com/oceanprotocol/pdr-backend/pull/1106/files

**[No additional systems needed]**
I do not believe this warrants any additional overhead/complexity by:
- using a build process for ppss
- using the callstack to retrieve previous ppss data
- using some sort of push/pop system to manage ppss
- using any sort of more advanced ppss implementation

thank you